### ISSUE TITLE
Firefox has problems when it wants to render issuable filter component 

### ISSUE THREAD

**Issue by dreis2211** [gitlabhq/gitlabhq#9243]
Hey,

we updated today to the newest version of GitLab, because we wanted the new UI that was introduced already some time ago (we were on 7.2 before). Although it is much better than before, we experience an issue in Firefox whenever a site is rendered that includes the issuable filter component.

![firefox_issue_filter_bug](https://cloud.githubusercontent.com/assets/6304496/7480769/34ee3c60-f36b-11e4-933c-814bdba0ad03.png)

The problem here is that the UsersSelect component is constructed more than once and therefore GitLab runs into this bug of the [select2](https://github.com/select2/select2) project. 
* Issue: https://github.com/select2/select2/issues/2837
* Fix: https://github.com/select2/select2/pull/2838

This fix is only in the newest release (4.x) of [select2](https://github.com/select2/select2) and not part of the [select2-rails](https://github.com/argerim/select2-rails) project that works currently with 3.x.

But, I suggest to clean that up and try to construct the UsersSelect component only once per page. If at some point there is something slow in this component, you will need to do that anyways. I already have a fix for that in our environment. I will create a PR for this tomorrow at the latest. I just wanted to give other users the information already that might experience this bug.

Cheers and keep up the good work!

Christoph
